src/libs/SidebarUtils.ts

```diff
@@ -381,7 +383,11 @@ function getOptionData({
     }
 
     const lastActorDisplayName = OptionsListUtils.getLastActorDisplayName(lastActorDetails, hasMultipleParticipants);
-    const lastMessageTextFromReport = OptionsListUtils.getLastMessageTextForReport(report, lastActorDetails, policy);
+
+    if (!lastMessageTextFromReport) {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
Can we use a different param name that we don't have to override? Seems easy to avoid suppressing a rule